packages/fxa-content-server/app/tests/spec/views/mixins/password-mixin.js

```diff
@@ -168,25 +180,27 @@ describe('views/mixins/password-mixin', function () {
         assert.equal($passwordEl.attr('autocorrect'), null);
 
         // Ensure the show password state stays in sync
-        const $showPasswordEl = $passwordEl.siblings('.show-password');
+        const $showPasswordEl = $('#show-password');
         assert.isFalse($showPasswordEl.is(':checked'));
       });
 
       it('getAffectedPasswordInputs - gets all affected inputs', function () {
         $('#container').html(view.$el);
         let targets = view.getAffectedPasswordInputs('#show-password');
-        assert.lengthOf(targets, 1);
+        // Note that this affects 5 in the test template, but with the real template,
```

Comment:
I'm not super happy about this but that test template file was giving me all sorts of issues with the selectors I went with and mentioned in another comment. It functions as expected and what we really care about is that one more is targeted so this seemed okay until we refactor.